### DISCUSSION TITLE
Thanks all language json providers

### DISCUSSION THREAD

**Discussion by lllyasviel** [lllyasviel/Fooocus#1368] [Announcements]
We really appreciate your help in translation!

We need a bit more time to figure out how to merge and/or manage these files in the repo.

We are aware of your contributions and please wait a moment for a next step on our side!

See also: https://github.com/lllyasviel/Fooocus/discussions/758

**Comment by brych92** [comment]
HI, is there any Ukrainian language json?

**Reply by sXe92** [reply]
https://github.com/sXe92/Fooocus/blob/main/language/uk.json

**Reply by brych92** [reply]
Thanks!
